nasa/earthdata-search
# EDSC-3881: Create portal for CASEI

# Overview

### What is the feature?

Add a portal to EDSC for [CASEI](https://impact.earthdata.nasa.gov/casei/), the Catalog of Archived Suborbital Earth Science Investigations.

### What is the Solution?

A new portal was created using a query for the 107 projects/campaigns currently available from CASEI.

Note that if new projects are added to CASEI, we will need to update this portal config.

We have an old unused portal called [Suborbital](https://github.com/nasa/earthdata-search/tree/2b6b38b3f34c8af6c0cadd9e3f01470cad5fb374/portals/suborbital) that looks like it was intended to be use used for CASEI. I tested it, but its query returned fewer collections (669).

# Testing

- Click Browse Portals
- Ensure there is an option for CASEI, then click it to enter the portal
- There should be around 1,768 matching collections

### Attachments

CASEI added to the list of portals:

<img width="600" alt="image" src="https://github.com/nasa/earthdata-search/assets/10775395/7db8279d-aca1-4aa7-9bac-aab54392fe03">


After entering portal:

<img width="600" alt="image" src="https://github.com/nasa/earthdata-search/assets/10775395/61a70d1e-1e30-43aa-b829-50f153f63532">

# Checklist
